zephyr/CMakeLists.txt

```diff
@@ -235,29 +235,39 @@ zephyr_system_include_directories(../examples/platforms)
 # Determine which libs should be linked in
 set(ot_libs "")
 
+if(CONFIG_OPENTHREAD_COPROCESSOR_RCP)
+else()
 if(CONFIG_OPENTHREAD_FTD)
 set(cli_lib openthread-cli-ftd)
 elseif(CONFIG_OPENTHREAD_MTD)
 set(cli_lib openthread-cli-mtd)
 endif()
+endif()
 
 if(CONFIG_OPENTHREAD_SHELL)
 list(APPEND ot_libs ${cli_lib})
 endif()
 
-if(CONFIG_OPENTHREAD_NCP)
+if(CONFIG_OPENTHREAD_COPROCESSOR_RCP)
+list(APPEND ot_libs openthread-rcp)
+endif()
+
+if(CONFIG_OPENTHREAD_COPROCESSOR_NCP)
 if(CONFIG_OPENTHREAD_FTD)
 list(APPEND ot_libs openthread-ncp-ftd)
 elseif(CONFIG_OPENTHREAD_MTD)
 list(APPEND ot_libs openthread-ncp-mtd)
 endif()
 endif()
 
+if(CONFIG_OPENTHREAD_COPROCESSOR_RCP)
```

Comment:
Let's make it `if (NOT CONFIG_OPENTHREAD_COPROCESSOR_RCP)`

Reply:
Applied.